python/ray/autoscaler/updater.py

```diff
@@ -153,16 +156,18 @@ def do_update(self):
                         cmd,
                         ssh_options_override=SSHOptions(
                             self.auth_config.get("ssh_private_key")))
+                if isinstance(self.cmd_runner, DockerCommandRunner):
+                    self.cmd_runner.run_init(self.initialize_as_head)
 
             with LogTimer(
                     self.log_prefix + "Setup commands", show_status=True):
                 for cmd in self.setup_commands:
-                    self.cmd_runner.run(cmd)
+                    self.cmd_runner.run(cmd, run_env="docker")  # TBD TODO(ilr)
```

Comment:
what are these TODO pending on?

Reply:
This is leftover from debugging. This was initially `run_env="host"` for some arcane reason